openadapt/utils.py

```diff
@@ -26,16 +26,31 @@ def configure_logging(logger, log_level):
     log_level_override = os.getenv("LOG_LEVEL")
     log_level = log_level_override or log_level
     logger.remove()
-    logger.add(sys.stderr, level=log_level)
+    logger.add(
+        sys.stderr,
+        level=log_level,
+        filter=filter_log_messages if config.IGNORE_WARNINGS else None,
+    )
     logger.debug(f"{log_level=}")
 
 
+def filter_log_messages(data):
+    messages_to_ignore = [
```

Comment:
This seems like it's from another PR, is that right? Fine to keep it here but should prioritize getting that merged 🙏 

Also, what do you think about moving this to `config.py`?

Reply:
Yeah I accidentally let it through since I started a new branch based off that, but it's ready for merging anyway. I agree, I think it makes more sense in config.py